ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -164,7 +164,11 @@
                      (api-core/get-user-id-from-token context)
                      subscriber)
         email (if-not email
```

Comment:
use `(if email email ....)` over `(if-not email ... email)`

Reply:
👍